Libraries/Components/WebView/WebView.ios.js

```diff
@@ -118,6 +118,15 @@ var WebView = React.createClass({
      */
     bounces: PropTypes.bool,
     /**
+     * Used on iOS only. Setting this to true enables "momentum" scrolling
+     * (default iOS scroll behavior) on the WebView.
+     * default value `false`
+     * **NOTE** By default WebView sets a faster deceleration rate.
+     * Enabling this may cause rendering performance to suffer.
+     * @platform ios
+     */
+    decelerationNormalEnabled: PropTypes.bool,
```

Comment:
Can you try to find a better name for this. It doesn't read as English to me nor does it really convey what it is doing.